WordPressAuthenticator/Unified Auth/View Related/Reusable Views/TextFieldTableViewCell.swift

```diff
@@ -52,23 +61,96 @@ private extension TextFieldTableViewCell {
     func setCommonTextFieldStyles() {
         textField.font = UIFont.preferredFont(forTextStyle: .body)
         textField.autocorrectionType = .no
-        textField.returnKeyType = .continue
     }
 
     /// Sets the textfield keyboard type and applies common traits.
     /// - note: Don't assign first responder here. It's too early in the view lifecycle.
     ///
     func applyTextFieldStyle(_ style: TextFieldStyle) {
-        switch style {
+		switch style {
         case .url:
             textField.keyboardType = .URL
-        default:
-            setCommonTextFieldStyles()
+			textField.returnKeyType = .continue
+		case .username:
+			textField.keyboardType = .default
+			textField.returnKeyType = .next
+		case .password:
+			textField.keyboardType = .default
+			textField.returnKeyType = .continue
+			setSecureTextEntry(true)
+			showSecureTextEntryToggle = true
+			configureSecureTextEntryToggle()
         }
     }
 }
 
 
+// MARK: - Secure Text Entry
+/// Methods ported from WPWalkthroughTextField.h/.m
+///
+private extension TextFieldTableViewCell {
+
+	/// Build the show / hide icon in the textfield.
+	///
+	func configureSecureTextEntryToggle() {
+		if showSecureTextEntryToggle == false {
+			return
+		}
```

Comment:
Just a suggestion - maybe this could be a `guard`?

```
guard showSecureTextEntryToggle else {
	return
}	
```

Reply:
Thanks! I converted the Obj-C class `WPWalkthroughTextField` into Swift and forgot to make the code actually Swift-y 😀